db/migrate/20170411000854_add_state_to_order.rb

```diff
@@ -0,0 +1,5 @@
+class AddStateToOrder < ActiveRecord::Migration[5.0]
+  def change
+    add_column :orders, :state, :string
```

Comment:
@Rdore88 I don't see any call to `order.state`  I do see a fetch on the json response from the production server.  But I am not sure if I follow.